### ISSUE TITLE
ara 1.5.5 hangs during installation

### ISSUE THREAD

**Issue by hille721** [ansible-community/ara#236]
## What is the issue ?
Direct installation from ara version 1.5.5 via pip hangs:
```
$ pip install ara
Collecting ara
  Cache entry deserialization failed, entry ignored
  Downloading .../pypi/packages/packaa496030eab77073b2b2ff94b61a1108/ara-1.5.5.tar.gz (2.5MB)
    100% |████████████████████████████████| 2.5MB 525kB/s 
```
Waiting for minutes and still nothing happend. I see following proceess:
```
python3 -c import setuptools, tokenize;__file__='/tmp/pip-build-16wqq3gv/ara/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec')) egg_info --egg-base pip-egg-info
```

Installing ara version 1.5.4 and then upgrading to 1.5.5 works without problems. Really funny.

tested OS and Python Version:
* RedHat 7, RedHat 8
* Python 3.6, Python 3.8

## What should be happening ?
ara version 1.5.5 should be also be installable as 1.5.4

**Comment by dmsimard** [comment]
@hille721 I haven't come across this before and just tested that it installs just fine in a venv under Fedora 33 and CentOS 8 on two different networks. Are you able to reproduce it elsewhere ? I'm wondering if there's an underlying network or system condition causing this.

**Comment by hille721** [comment]
Yes you're right, I have the issues only at my company network where we have a mirrored PyPi index. Maybe the package there is somehow corrupted.

Will close the issue.

**Comment by hille721** [comment]
Hi @dmsimard,

I reopen this issue, because with 1.5.6 I have the same problem. 
Installing 1.5.4 from our company artifactory and then upgrading to 1.5.5 or 1.5.6 is working without any problems, but not directly install it.

I now also figured out that you have stopped to build the wheel binaray packages. Is there a reason why? For 1.5.4 there was also a wheel available and I just checked that this what our artifactory is using. 
Maybe also a wheel for the newer version would already fix my problem.

**Comment by hille721** [comment]
aaah, I', such a fool! Upgading setuptools fixes the problem!

My setuptools version:
```
setuptools 41.6.0 from venv/lib/python3.8/site-packages (Python 3.8)
```

Upgrading to `setuptools-56.0.0` and everything works :)

**Comment by dmsimard** [comment]
> I now also figured out that you have stopped to build the wheel binaray packages. Is there a reason why?

An oversight coming from when we migrated from gerrit+openstack zuul to github+ansible zuul. In a nutshell, releases were handled automatically by Zuul which also took care of building the wheel. I have not gotten around to implementing this back since the migration so 1.5.5 and 1.5.6 were released "manually" without realizing the wheels were not built and published.

I just created an issue so we don't forget about it: https://github.com/ansible-community/ara/issues/271
